docs/docs/getting-started/demo/without-docker.md

```diff
@@ -45,12 +45,13 @@ cabal exec cardano-node -- run \
 
 ## Seeding The Network
 
-You can use the `seed-devnet.sh` script by passing it the path to a cardano-cli executable to use, instead of having it using the Docker container. For example:
+You can use the `seed-devnet.sh` script by passing it the path/command to a cardano-cli and hydra-node executable to use, instead of having it using the Docker container. For example:
 
 
 ```mdx-code-block
 <TerminalWindow>
-./seed-devnet.sh $(which cardano-cli)
+export CARDANO_NODE_SOCKET_PATH=devnet/ipc/node.socket
+./seed-devnet.sh $(which cardano-cli) "cabal exec hydra-node --"
```

Comment:
That is slightly weird :thinking: ... Why not `$(which hydra-node)` too for consistency? I mean, this boils down to the same thing given how it is invoked but, I don't know, the example feels off to me :sweat_smile:

Reply:
It is kind of highlighting both ways. The `hydra-node` executable would only be in scope if one would be using the `nix-shell -A demo` environment and below we also use `cabal exec` invocations. Not sure which way is more clear :shrug: